frontend/src/components/3.Preferences/Preferences.js

```diff
@@ -53,7 +53,7 @@ function Preferences() {
   async function handleSearch() {
     // console.log(Coordinates);
     const data = await getNearbyRestaurants(
-        Coordinates, Distance, 'european');
+        Coordinates, Distance, 'european', Price);
```

Comment:
I noticed that the `<option>` tags in this file on lines 162-165 set string values for the prices. Does the API call set the price correctly if the maxPriceLevel is set to a string value?

Maybe it would be better to change it? So the option values would be numbers, e.g. `<option value={1}>` instead of `<option value='1'>`

Reply:
I see what you meant. I agree with you after changing that the type of value making more sense.